src/examples/avatar/AvatarShape.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as ZendeskIcon } from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+import { ReactComponent as UserStrokeIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin-bottom: ${p => p.theme.space.sm};
+  text-align: center;
```

Comment:
Since there is such an appetite for this, we should probably add a `textAlign` prop to the grid `<Col>` component. With this and the comment above, a lot of the `StyledCol` usage in the website becomes redundant.

Reply:
These styles are really dependent on content. I'm not sure we can accurately say how all content could be handled in RTL for example.  Should `text-align: left` always go to `right` in RTL? And vice-versa for other styles?

Reply:
With the `Grid` updates being released my comment is now out-of-date 😄

Reply:
A new commit updates the `react-grid` package and leverages the `textAlign` prop on `Col`.